docs/src/index.md

```diff
@@ -23,6 +23,24 @@ ftarget = DA(zeros(FT, ntarget)) # allocated function on target grid
 interpolate1d!(ftarget, xsource, xtarget, fsource, Linear(), Flat())
 ```
 
+In-place interpolation is supported on reverse (monotonically decreasing grids).
```

Comment:
Could you add a note here that `xsource` and `xtarget` must have the agree, i.e. that they must either both be increasing  (and `reverse = true` should be used), or both be decreasing (and `reverse = true` should be used)? I know it's mentioned in the source code, but it would be nice for the docs to be more standalone as a resource